README.md

```diff
@@ -1,2 +1,93 @@
 # drf-madprops
+
 DRF library of helpers to operate lists of resources' properties as dicts
```

Comment:
The single quote is correct, properties belonging to many resources. However I believe it would still be correct as `lists of resource properties`.

The phrase `to operate lists` is bothering me. Maybe you mean `to operate on lists`?